PokeDemo/Extensions/UIView.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  UIView.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 03/12/22.
+//
+
+import UIKit
+
+extension UIView {
```

Comment:
why it is required?

Reply:
Added IBInspectable attributes here for views.

Reply:
Why?